api/src/badge_logic/get_badge_info.js

```diff
@@ -70,7 +70,7 @@ function simpleDateComp (requiredDay, days, operator, today, excluded) {
   const validDate = new Date(validDay[0], Number(validDay[1] - 1), validDay[2])
   if (!(execLogic(operator, today, validDate))) {
     return [ pass, true, '' ]
-  } else if (operator[0] === '<') {
```

Comment:
I was having operators other than `<` hit this and because they were unhandled this function returned `undefined`.

Reply:
The idea behind that was that the first condition checks to see whether mapping today would not contribute to the badge, and if it would contribute to the badge, then the "due by" date must be in the future.

I'm assuming the operator that hit the else was "=" when you were testing with validDate = today? I think it makes sense to provide for that, but if other operators got through then something else may be going on.